ingestion/src/metadata/ingestion/source/database/dbt_source.py

```diff
@@ -96,14 +97,25 @@ def _parse_data_model(self):
                     schema = mnode["schema"] if mnode["schema"] else "default"
                     raw_sql = mnode.get("raw_sql", "")
                     description = mnode.get("description")
-                    dbt_user_name = cnode["metadata"].get("owner")
-                    user_name = f"*{dbt_user_name}*"
-                    user_fqn = fqn.build(
-                        self.metadata, entity_type=User, user_name=user_name
-                    )
-                    owner = self.metadata.get_entity_reference(
-                        entity=User, fqn=user_fqn
-                    )
+                    dbt_owner = cnode["metadata"].get("owner")
+                    owner = None
+                    if dbt_owner:
+                        owner_name = f"*{dbt_owner}*"
```

Comment:
@OnkarVO7 what will be the scenario when the owner name is `John`

and we have 3 users named 
`John`, `JohnDoe`,`JaneDoe`

will it take the first one?
if so, do we need quotes around this? like a wildcard search? or just an exact match should be fine?